components/centraldashboard/public/components/activity-view.js

```diff
@@ -1,112 +1,109 @@
-import {PolymerElement, html} from '@polymer/polymer';
-
 import '@polymer/iron-flex-layout/iron-flex-layout-classes.js';
 import '@polymer/iron-ajax/iron-ajax.js';
 import '@polymer/iron-icon/iron-icon.js';
 import '@polymer/iron-icons/iron-icons.js';
 import '@polymer/paper-progress/paper-progress.js';
 
+import {html, PolymerElement} from '@polymer/polymer';
+import './activities-list.js';
+
 export class ActivityView extends PolymerElement {
     static get template() {
         return html`
             <style is="custom-style" include="iron-flex iron-flex-alignment">
             </style>
             <style>
                 :host {
-                    @apply --layout-vertical;
-                    background: #f1f3f4;
+                    @apply --layout-fit;
                     --accent-color: #007dfc;
                     --primary-background-color: #003c75;
                     --sidebar-default-color: #ffffff4f;
                     --border-color: #f4f4f6;
+                    background: #f1f3f4;
+                    min-height: 50%;
```

Comment:
`Min-Height` not needed, and `overflow: auto` is missing

Reply:
Thanks for catching!